packages/treasury/test/test-stablecoin.js

```diff
@@ -308,67 +410,33 @@ test('first', async t => {
 });
 
 test('price drop', async t => {
-  /* @type {TestContext} */
-  let testJig;
-  const setJig = jig => {
-    testJig = jig;
-  };
-  const { zoe, feeMintAccess } = await setUpZoeForTest(setJig);
-
-  const autoswapInstall = await makeInstall(autoswapRoot, zoe);
-  const stablecoinInstall = await makeInstall(stablecoinRoot, zoe);
-  const liquidationInstall = await makeInstall(liquidationRoot, zoe);
-
   const {
     aethKit: { mint: aethMint, issuer: aethIssuer, brand: aethBrand },
   } = setupAssets();
 
-  const priceAuthorityPromiseKit = makePromiseKit();
-  const priceAuthorityPromise = priceAuthorityPromiseKit.promise;
+  const manualTimer = buildManualTimer(console.log);
```

Comment:
I tried to get some of these timers to be quiet unless `ava` sees `-v` by doing...

```suggestion
  const manualTimer = buildManualTimer(t.log);
```

but sometimes `t` is not in scope... I'm not sure whether it's worth doing just some of the time.

Reply:
What problem are you trying to solve?

Reply:
The the problem of lots of unwanted console logs when running tests.

The `t.log` function is silent unless the `-v` command-line flag is given.